### PR TITLE
feat: enable showing jobs in `e ci status`

### PR DESCRIPTION
**Pull request by codebytere**
Enable showing individual job status and job IDs for individual checks in `e ci status`.

```
build-tools on git:ci-status-show-jobs ❯ e ci status -s                    10:08AM
Electron CI Status
  Ref: main

  Circle CI
  ⦿ macOS - failed - https://circleci.com/workflow-run/72f39358-9341-4a20-b36a-6220789d44b0
     ⦿ success mac-make-src-cache, mas-testing-arm64, mas-testing-arm64-tests, mas-testing-x64, mas-testing-x64-tests, mas-testing-x64-gn-check, osx-testing-arm64, osx-testing-arm64-tests, osx-testing-x64, osx-testing-x64-gn-check
     ⦿ failed - osx-testing-x64-tests - 636701d1-19f4-484d-b23b-995378041905

  ⦿ Linux - success - https://circleci.com/workflow-run/636344b2-ebd4-49c2-95ce-631ecd548b73
     ⦿ all jobs succeeded

  ⦿ Lint - success - https://circleci.com/workflow-run/f3e7db2a-2734-4161-a529-535a25726098
     ⦿ all jobs succeeded


  Appveyor
  ⦿ Windows x64 - Success - https://ci.appveyor.com/project/electron-bot/electron-ljo26/builds/43857449

  ⦿ Windows x64 (PR) - Missing

  ⦿ Windows ia32 - Success - https://ci.appveyor.com/project/electron-bot/electron-ia32-testing/builds/43857450

  ⦿ Windows ia32 (PR) - Missing

  ⦿ Windows Arm - Success - https://ci.appveyor.com/project/electron-bot/electron-ldhmv/builds/43857451
```

### CODE REVIEW

**Review by MarshallOfSound — COMMENTED** [review]
Can we print the job names instead of their IDs, the ID isn't super helpful.

**Comment by codebytere** [comment]
@MarshallOfSound the job IDs are a precursor to being able to rerun jobs with `e ci rerun` - https://circleci.com/docs/api/v2/#operation/listWorkflowJobs there's no name per se here that i can find, do you have an alternative pref?

**Comment by MarshallOfSound** [comment]
Each of those job objects has a name property per those docs

**Comment by codebytere** [comment]
whoops i think i missed it bc it looked so weird in the example - done!

**Review by ckerr — APPROVED** [review]
Possible code style issue in a couple of lines; otherwise, looks very nice. I like this feat!

**Comment by MarshallOfSound** [comment]
We could also do cool stuff and detect compatible terminal apps and make the job names links 🤔 https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda

File: src/ci/ci-status.js [hunk 1]
```diff
@@ -65,52 +111,67 @@ program
...
+      }
+
       console.log(`${chalk.bold('Electron CI Status')}
   ${chalk.bold('Ref')}: ${chalk.cyan(ref)}
 
   ${chalk.bold(chalk.bgMagenta(chalk.white('Circle CI')))}
-  ${checkLine(macOS, 'macOS')}
-  ${checkLine(linux, 'Linux')}
-  ${checkLine(lint, 'Lint')}
+${printChecks(checks)}
```
[the remark this answers is not in the document]
Comment (codebytere) [reply src/ci/ci-status.js:171]: this is intentional! it is handled in the checks and status printer functions respectively

**Comment by codebytere** [comment]
@MarshallOfSound critical to this PR is the IDs - they're necessary for adding the ability to rerun single jobs in my next follow-up to https://github.com/electron/build-tools/pull/380. We can try to collapse some and perhaps not show IDs for the passed builds?

**Comment by MarshallOfSound** [comment]
Maybe job name _and_ id for failed builds and just the name for successful ones collapsed like a sketched out above

**Pull request merged**